Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Subsequences/Configurator.mo

```diff
@@ -0,0 +1,231 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Subsequences;
+block Configurator "Configures chiller staging"
+
+  parameter Integer nSta = 3
+    "Number of chiller stages";
+
+  parameter Integer nChi = 2
+    "Number of chillers";
+
+  parameter Modelica.SIunits.Power chiDesCap[nChi]
+    "Design chiller capacities";
+
+  parameter Modelica.SIunits.Power chiMinCap[nChi]
+    "Chiller unload capacities";
+
+  parameter Integer chiTyp[nChi]={
+    Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Types.ChillerTypes.positiveDisplacement,
+    Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Types.ChillerTypes.variableSpeedCentrifugal}
+    "Chiller type";
+
+  parameter Integer staMat[nSta, nChi] = {{1,0},{0,1},{1,1}}
+    "Staging matrix with stage as row index and chiller as column index";
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uChiAva[nChi]
+    "Chiller availability status vector"
+    annotation (Placement(transformation(extent={{-260,-20},{-220,20}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yAva[nSta]
+    "Stage availability status vector"
+    annotation (Placement(transformation(extent={{220,-50},{240,-30}}),
+      iconTransformation(extent={{100,-80},{120,-60}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerOutput yTyp[nSta](
+    final max=fill(nSta, nSta)) "Chiller stage types vector"
+    annotation (Placement(transformation(extent={{220,-90},{240,-70}}),
+      iconTransformation(extent={{100,-40},{120,-20}})));
+
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput yDesCap[nSta](
+    final unit=fill("W", nSta),
```

Comment:
Can we use `each` here? so it becomes `each final unit="W"`

Reply:
I think the last time I asked Michael mentioned that CDL does not support it. It may be worth checking again.

Reply:
Let's use `fill`. The `each` keyword is not supported (for now), see
http://obc.lbl.gov/specification/cdl.html#permissible-data-types